scene/gui/check_box.cpp

```diff
@@ -75,14 +66,30 @@ Size2 CheckBox::get_minimum_size() const {
 	Size2 tex_size = get_icon_size();
 	minsize.width += tex_size.width;
 	if (get_text().length() > 0) {
-		minsize.width += MAX(0, get_theme_constant(SNAME("h_separation")));
+		minsize.width += MAX(0, theme_cache.h_separation);
 	}
-	Ref<StyleBox> sb = get_theme_stylebox(SNAME("normal"));
-	minsize.height = MAX(minsize.height, tex_size.height + sb->get_margin(SIDE_TOP) + sb->get_margin(SIDE_BOTTOM));
+	minsize.height = MAX(minsize.height, tex_size.height + theme_cache.normal_style->get_margin(SIDE_TOP) + theme_cache.normal_style->get_margin(SIDE_BOTTOM));
 
 	return minsize;
 }
 
+void CheckBox::_update_theme_item_cache() {
+	Button::_update_theme_item_cache();
+
+	theme_cache.h_separation = get_theme_constant(SNAME("h_separation"));
+	theme_cache.check_v_adjust = get_theme_constant(SNAME("check_v_adjust"));
+	theme_cache.normal_style = get_theme_stylebox(SNAME("normal"));
```

Comment:
To be consistant with `Button` and the actual theme item name it should likely be `normal`.

Reply:
Or the other way around maybe, the `Button` ones could get `_style` suffix. I suppose you're already planning to harmonize all this and make theme item names more explicit.

Reply:
I'd prefer for the cache items to be explicit at least, so that the code that uses it is clear. Admittedly, in this PR I didn't update everything to a better style, but that's in part because indeed I can do that while harmonizing the names.